lib/cli/src/frameworks/react/js/0-Welcome.stories.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { linkTo } from '@storybook/addon-links';
-import { Welcome } from '@storybook/react/demo';
+import Welcome from './Welcome';
```

Comment:
Any rationale for default vs named imports? I ask bc we're standardizing on named imports in Storybook and apparently a lot of people feel passionately about that.

Reply:
I'm an export default person :D no worries of changing it